### DISCUSSION TITLE
1974-03-03-Solid-Gold-Sixty-with-Tom-Browne/

### DISCUSSION THREAD

**Discussion by giscus** [FanzOfLenaZavaroni/fanzoflenazavaroni.github.io#10] [General]
# 1974-03-03-Solid-Gold-Sixty-with-Tom-Browne/

Lena Zavaroni’s single Ma! (He’s Making Eyes At Me) moved up the charts from number 13 to number 10.

https://fanzoflenazavaroni.github.io/1974-03-03-Solid-Gold-Sixty-with-Tom-Browne/

**Comment by StephenMolloy1** [comment]
It seems that the Soundcloud uploads of the top 20 for this period are using the wrong Radio 1 Logo. The one used in 1973 - 75 was:
http://ftpmirror.your.org/pub/wikimedia/images/wikipedia/fr/3/32/BBC_Radio_1logo_1974.gif

**Reply by FanzOfLenaZavaroni** [reply]
Thanks for the link to the correct image related to the dates we are interested in.

I'm not surprised that the thumbnails used are wrong some of the files use an image of the artist who was at number 1 that week but may not be of how the artist looked in that broadcast year. The individuals uploading just needs a thumbnail to be added to their file and most likely went with an image near the top of any image search they may had done for the Radio 1 logo.

I'm more concerned that this episode of the Top 20 was taken down and hope there are not more removals coming as The Wayback Machine doesn't seem to archive Mixcloud content.

I had downloaded the handful of episodes I had found on there and will check to see if there are others I need to get before any possible take down from Mixcloud it's a bit of a disappointing website in someways for a podcasting platform as it has no RSS feed to just add it to a podcasting app for downloading.

So I had to manually download the few I got manual, well semi-manually as I had to copy and pasted the URLs for each episode into a text file and then run a script to read and then download the audio for each URL.

What I really need to do is find the time to listen to the episodes and make a note of the artist(s), song and chart position so that I can then add that info. to each of the related pages on this website.

While writing this way to long reply I just realised I could just use the details from the "Record and Radio Mirror", I'll need to check but I think they use the same chart data. If they do use the same data then I'll also be able to create pages for broadcasts of Solid Gold Sixty with Tom Browne even if I can't locate audio files of them.

**Reply by StephenMolloy1** [reply]
I've put it on my You Tube Channel to try get away from the reliance on Mixcloud. Like you, I will try to get all of Lena's appearances put up on You Tube, but as you said - it isn't straightforward.
